lib/runner.js

```diff
@@ -46,6 +46,10 @@ var Runner = function(config) {
     flow.timeout(1000, 'waiting for debugger to attach');
   }
 
+  if (config.debuggerServerPort) {
+    process.debuggerServerPort = config.debuggerServerPort;
```

Comment:
I'd prefer to do something a bit roundabout, but explicit, instead of adding properties to `process` randomly. Can we add a member in Protractor and just change it directly (e.g. `browser.debugPort = 8798`)